src/utils/event/event.ts

```diff
@@ -419,4 +475,29 @@ export const eventMap: EventMap = {
     EventType.TestLzEvent,
     'LzEvent(uint16 _dstChainId, bytes _destination, bytes _payload)',
   ),
+  [EventType.MintFeePayout]: eventBuilder(
+    EventType.MintFeePayout,
+    'MintFeePayout(uint256 mintFeeAmount, address mintFeeRecipient, bool success)',
```

Comment:
Do these event signatures work as they are? Typically, the argument names are omitted to get event signature

Reply:
in my local tests, they worked normally